From: Jens Reidel <adrian@mainlining.org>
Date: Sun, 21 Sep 2025 10:09:31 +0200
Subject: dt-bindings: regulator: qcom,sdm845-refgen-regulator: document
 SM7150

Document the refgen clock being present on the SM7150 platform. It is
used to provide reference voltage to the DSI controller.

Signed-off-by: Jens Reidel <adrian@mainlining.org>
---
 .../devicetree/bindings/regulator/qcom,sdm845-refgen-regulator.yaml      | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/regulator/qcom,sdm845-refgen-regulator.yaml b/Documentation/devicetree/bindings/regulator/qcom,sdm845-refgen-regulator.yaml
index 40f9223..f3e81de 100644
--- a/Documentation/devicetree/bindings/regulator/qcom,sdm845-refgen-regulator.yaml
+++ b/Documentation/devicetree/bindings/regulator/qcom,sdm845-refgen-regulator.yaml
@@ -24,6 +24,7 @@ properties:
               - qcom,sc7180-refgen-regulator
               - qcom,sc8180x-refgen-regulator
               - qcom,sdm670-refgen-regulator
+              - qcom,sm7150-refgen-regulator
               - qcom,sm8150-refgen-regulator
           - const: qcom,sdm845-refgen-regulator
 
